src/test/scala/uk/gov/hmcts/ccd/corecasedata/scenarios/ValidateCaseDetails.scala

```diff
@@ -0,0 +1,77 @@
+package uk.gov.hmcts.ccd.corecasedata.scenarios
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import uk.gov.hmcts.ccd.util.{CcdTokenGenerator, PerformanceTestsConfig}
+
+import scala.concurrent.duration._
+
+object ValidateCaseDetails extends PerformanceTestsConfig {
```

Comment:
All naming in this class is completely messed up. You're not Validating a case here, you are creating a case.
Class name wrong, urls names misleading 
Please review carefully and fix

Reply:
Reviewed and all looks OK to me
that's validateCaseDetails test and working perfectly fine.

Object name ValidateCaseDetails is also looking fine to me.